packages/rrweb/test/__snapshots__/replayer.test.ts.snap

```diff
@@ -8,12 +8,6 @@ exports[`replayer can fast forward past StyleSheetRule changes on virtual elemen
   </head>
   <body>
     <div class=\\"replayer-wrapper\\">
-      <canvas
```

Comment:
the mousetail canvas is no longer getting created automatically so removing these from the snapshot